parts/k8s/addons/kubernetesmasteraddons-managed-azure-storage-classes.yaml

```diff
@@ -10,6 +10,7 @@ provisioner: kubernetes.io/azure-disk
 parameters:
   kind: Managed
   storageaccounttype: Standard_LRS
+  cachingmode: None
```

Comment:
What is the default cachingmode if it is not set?

Reply:
default is `ReadWrite`, and I have set to `None` in current k8s upstream code from v1.7-v1.10, all minor version should fix this issue.